setup.py

```diff
@@ -8,6 +8,8 @@
 
 deps = [
     'marionette-client == 0.9',
+    'mozinstall == 1.11',
+    'mozlog == 2.10',
```

Comment:
One other benefit would be that our package will not be broken if a new release of mozlog has a regression in some part. We can clearly control which version of dependencies we want to have.